site/en/docs/privacy-sandbox/topics/demo/index.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Topics API demos'
+subhead: >
+  Try out the Topics API demos. 
+description: >
+  Try out the Topics API demos.
+date: 2022-01-25
+updated: 2022-02-01
+authors:
+  - samdutton
+---
+
+The Topics API demo provides a look at how topics are inferred from hostnames. You can preview what topics are observed when you visit a demo site, which requires very little setup.
+
+If you want to test the API with your users, sign up for the [Relevance and Measurement origin trial](LINK).
+
+Our demo is a preview  that demonstrates most features of the Topics API, for you to gain familiarity with how the API is implemented.
+
+The demo of the Topics API  is at [topics-demo.glitch.me](https://topics-demo.glitch.me/).
+This explains how to try out and debug the API for a single user.
```

Comment:
```suggestion
```

Since there are two demos, which are both linked below, let's skip introducing it in the introduction.